kitsune/sumo/static/tpl/search-results.html

```diff
@@ -28,13 +28,13 @@
     {% endif %}
     <ul id="product-filter" class="search-filter sidebar-nav">
       <li {{ product|class_selected(null)|safe }}>
-        <a href="#" data-instant-search-unset-params="product" data-instant-search="link" data-href="{{ base_product_url }}">
+        <a href="#" data-instant-search-unset-params="product" data-instant-search-set-params="all_products=1" data-instant-search="link" data-href="{{ base_product_url|urlparams(all_products=1) }}">
```

Comment:
It seems like you specify `all_products=1` twice here, once in `data-instant-search-set-params` and once in `data-href`. Are the both needed?